frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -206,7 +210,28 @@ export const NavBar = withRouter<NavBarProps>(({ pages, baseURL, basePath }) =>
     </>
   );
 
-  return <ul className="co-m-horizontal-nav__menu">{tabs}</ul>;
+  const activePage = pages.find(({ href, path }) => {
+    const matchURL = matchPath(location.pathname, {
+      path: `${basePath}/${removeLeadingSlash(path || href)}`,
+      exact: true,
+    });
+    return matchURL?.isExact;
+  });
+
+  const labelId =
+    (activePage?.nameKey && activePage?.nameKey.split('~')[1]) || activePage?.name || 'Details';
+  return (
+    <>
+      {titlePrefix && (
```

Comment:
Is there a reason to not set the current activePage.name as title when there is no `titlePrefix`?

Couldn't we set this always. But you're right we should expect that the `titlePrefix is not defined.